src/common/api.js

```diff
@@ -103,7 +104,8 @@ exports.timeline = function (user, options, callback) {
         try {
             return JSON.parse(str);
         } catch(e) {
-            cli.log.error('Parsing error:' + str + '\n');
+            if(str!="")
```

Comment:
@victorperin please make the code style similar to what we have in repo, also add strict comparison and single quotes:

```javascript
if (str !== '') {
   cli.log.error('Parsing error:' + str + '\n');
}
```